src/App.tsx

```diff
@@ -28,7 +30,9 @@ const App = () => {
       <Suspense fallback={<Skeleton />}>
         <Routes>
           <Route path="/" element={<Dashboard />} />
+          <Route path="/envoy-fleets" element={<EnvoyFleets />} />
```

Comment:
Wouldn't it be better to have an array of routes? Mark as resolve, please, if you don't agree